jujugui/static/gui/src/app/components/import-export/_import-export.scss

```diff
@@ -29,6 +30,8 @@
   }
 
   &__file {
+      position: absolute;
+      top: -99999;
```

Comment:
Drive by to move the file input off screen so it doesnt block clicks in the header.